RubeesEat/Pages/BillDetails.cshtml

```diff
@@ -0,0 +1,28 @@
+@page "{guid}"
+@using System.Globalization
+@inject IBillRepository billRepository
+
+Bill Details:
+@{ var item = billRepository.GetById(Guid.Parse(RouteData.Values["guid"].ToString())); }
+@if (item == null)
+{
+    <div id="errorText" style="color:red"> Bill was not found!</div>
+}
+else
+{
+    <div class="personWhoPayed" style="">Paid by: @(item.EntryLines[0].Person.FirstName) </div>
+    <div class="date">@item.Date.ToString("D")</div>
+    <div class="description">@item.Description</div>
+    <div class="amount" style="display: inline;">Total amount: @item.EntryLines[0].Amount€</div>
+    <ul>
+        @for (int i = 1; i < item.EntryLines.Length; i++)
+        {
+            <li id="entryLine">
+
+                <div class="person" style="display:inline;">@(item.EntryLines[i].Person.FirstName): </div>
+                <div class="amountPerPerson" style="display:inline; color: @(item.EntryLines[i].Amount < 0 ? "red" : "green")">@item.EntryLines[i].Amount€ </div>
+
+            </li>
+        }
+    </ul>
+}
```

Comment:
To stay consistent with split a bill, please always use FirstName LastName (don't forget tests)